src/main/java/org/openjdk/jextract/clang/Index.java

```diff
@@ -98,6 +115,17 @@ public TranslationUnit parseTU(String file, Consumer<Diagnostic> dh, int options
         }
     }
 
+
+    public TranslationUnit parseTU(String file, Consumer<Diagnostic> dh, int options, String... args)
```

Comment:
Can't the methods here that _don't_ take a file content be cleaned up as well now?

Reply:
It was used in macro parser. But with changes in macro parse implementation, these may not be. Will remove.